src/main/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentEntry.java

```diff
@@ -0,0 +1,159 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang.StringUtils;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = StringUtils.EMPTY, propOrder = {
+    "index",
+    "documentTypeCode",
+    "description",
+    "explanation",
+    "organizationDocumentNumber",
+    "sourceAccountingLines",
+    "targetAccountingLines",
+    "notes",
+    "adHocRecipients",
+    "backupLinks"
+})
+@XmlRootElement(name = "Document", namespace = StringUtils.EMPTY)
+public class AccountingXmlDocumentEntry {
+
+    @XmlElement(name = "Index", namespace = StringUtils.EMPTY, required = true)
+    protected Long index;
+
+    @XmlElement(name = "DocumentType", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String documentTypeCode;
+
+    @XmlElement(name = "Description", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String description;
+
+    @XmlElement(name = "Explanation", namespace = StringUtils.EMPTY, required = false)
+    protected String explanation;
+
+    @XmlElement(name = "OrganizationDocumentNumber", namespace = StringUtils.EMPTY, required = false)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String organizationDocumentNumber;
+
+    @XmlElementWrapper(name = "AccountingListFrom", namespace = StringUtils.EMPTY, required = false)
+    @XmlElement(name = "Accounting", namespace = StringUtils.EMPTY, required = false)
+    protected List<AccountingXmlDocumentAccountingLine> sourceAccountingLines;
+
+    @XmlElementWrapper(name = "AccountingListTo", namespace = StringUtils.EMPTY, required = false)
+    @XmlElement(name = "Accounting", namespace = StringUtils.EMPTY, required = false)
+    protected List<AccountingXmlDocumentAccountingLine> targetAccountingLines;

```

Comment:
should this be toAccountingLines

Reply:
Ok, I'll check the naming on this and change it accordingly.